pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopicPolicies.java

```diff
@@ -1729,6 +1743,48 @@ void run() throws PulsarAdminException {
                 }
             }
 
+            int maxBlockSizeInBytes = OffloadPoliciesImpl.DEFAULT_MAX_BLOCK_SIZE_IN_BYTES;
+            if (StringUtils.isNotBlank(maxBlockSizeInBytesStr)) {
+                long maxBlockSize = validateSizeString(maxBlockSizeInBytesStr);
+                if (positiveCheck("MaxBlockSizeInBytes", maxBlockSize)
+                        && maxValueCheck("MaxBlockSizeInBytes", maxBlockSize, Integer.MAX_VALUE)) {
```

Comment:
The same problem also exists in `CmdTopics`.
Bytes should use  long instead of int, right?